app/assets/stylesheets/utilities/layout.scss

```diff
@@ -9,3 +9,7 @@
 .justify-content-between {
   justify-content: space-between;
 }
+
+.small-table-button {
+  margin-bottom: 0px;
```

Comment:
There are govuk design system classes if you prefer to not write a custom one - have a look [here](https://design-system.service.gov.uk/styles/spacing/).